ci/ci/templates/pr-table.html

```diff
@@ -30,6 +31,11 @@
           *
         {% endif %}
       </td>
+      <td>
+        {% for context, status in pr['gh_statuses'].items() %}
+          {{ context }}: {{ status }}
```

Comment:
Should I be making rows instead here to stack these on top of each other? Not too familiar with jinja.

Reply:
The question is less  to do with ninja and more to do with HTML.

WRT stacking rows, do you want to achieve this?
```
+----------------------+
|         | azure: yes |
|  #12345 +------------+ 
|         | gcp: no    |
+----------------------+
```

Reply:
I don't think this is possible unless we fix the number of other CIs. If we fix the number of other CIs, we should just create two columns, one for azure and one for GCP.

I think that's probably the right thing to do here. Something like this
```python3
def get_gh_status_str(statuses, status_key):
    x = statuses(status_key)
    if x is None:
        return x
    return x.value
# ...
'gh_statuses': {get_gh_status_str(pr.last_known_github_status, k) for k in ('hail-ci', 'ci-test-azure')},
```

You could also dynamically figure out the number of distinct status across all PRs and create that many columns, etc. Either approach seems fine to me.